plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -980,6 +980,15 @@ void GSRendererDX11::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sou
 	}
 
 	m_ps_sel.fba = m_context->FBA.FBA;
+	m_ps_sel.dither = m_env.DTHE.DTHE;
+
+	if(m_ps_sel.dither)
```

Comment:
Maybe you can do if (m_env.DTHE.DTHE) instead and add m_ps_sel.dither = 1 inside the function.

Reply:
I prefer the lowercase letters. If I add ` m_ps_sel.dither = 1` inside the if, then I would need an else ` m_ps_sel.dither = 0` and kinda looks ugly.